src/entity.js

```diff
@@ -139,6 +139,7 @@ export class Entity {
     return this.getTransport()
       .create(this.getResource(), this.asObject(true))
       .then(created => {
+        if (created===null||created[this.getIdProperty()]===undefined) throw new Error('The API resource did not return the id property "'+this.getIdProperty()+'"');
```

Comment:
Please follow the code conventions. Spaces surrounding or statement and curly braces.